lib/plugins/aws/provider/awsProvider.js

```diff
@@ -233,6 +233,10 @@ class AwsProvider {
             UpdateReplacePolicy: { type: 'string' },
             Condition: { type: 'string' },
           },
+          awsLogGroup: {
+            type: 'string',
+            pattern: '^/aws/[a-zA-Z0-9-_.]+$',
```

Comment:
It doesn't seem to follow pattern in AWS docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-subscriptionfilter.html#cfn-cwl-subscriptionfilter-loggroupname